agent/agent/rules/conditions.py

```diff
@@ -0,0 +1,31 @@
+"""Rule condition helpers."""
+
+from arxiv.submission.domain.event import Event, AddFeature
+from arxiv.submission.domain.submission import Submission
+from arxiv.submission.domain.agent import Agent, User, System
+from arxiv.submission.domain.annotation import Feature
+
+from .base import Condition
+
+
+def system_event(event: Event, before: Submission, after: Submission) -> bool:
```

Comment:
```suggestion
def is_system_event(event: Event, before: Submission, after: Submission) -> bool:
```
per
https://confluence.cornell.edu/pages/viewpage.action?pageId=372900900

Reply:
Got me! Fixed predicate names in b6fb0c1